### PR TITLE
[rebranch] Fix rebranch tests

### PR DESCRIPTION
**Pull request by bnbarham**

### CODE REVIEW

File: test/SILGen/cf_curried_init.swift [hunk 1]
```diff
@@ -8,4 +8,4 @@ import CoreGraphics
 let _: (CFURL) -> CGDataProvider? = CGDataProvider.init
 
 // CHECK-LABEL: sil private [ossa] @$s15cf_curried_initSo17CGDataProviderRefaSgSo8CFURLRefacfu_ : $@convention(thin) (@guaranteed CFURL) -> @owned Optional<CGDataProvider>
-// CHECK-LABEL: sil [available {{.*}}] [clang CGDataProvider.init] @CGDataProviderCreateWithURL : $@convention(c) (CFURL) -> @owned Optional<CGDataProvider>
+// CHECK-LABEL: sil [clang CGDataProvider.init] @CGDataProviderCreateWithURL : $@convention(c) (CFURL) -> @owned Optional<CGDataProvider>
```
Comment (bnbarham) [inline test/SILGen/cf_curried_init.swift:11]: @tshortli any idea what could have caused this? As far as I can tell this test was added to make sure it doesn't crash, but it seems odd that the availability was removed.

**Comment by bnbarham** [comment]
@swift-ci please test

File: test/SILGen/cf_curried_init.swift (hunk 1 above)
Comment (tshortli) [reply test/SILGen/cf_curried_init.swift:11]: The availability info would be coming from the ClangImporter component in this case - maybe something has changed there on `rebranch`?

Comment (bnbarham) [reply test/SILGen/cf_curried_init.swift:11]: It failed in the toolchain build I used to test some llbuild changes (https://ci.swift.org/job/swift-PR-toolchain-macos/185). The regular jobs should get through llbuild on their next run so I can see if it happens there.

Comment (bnbarham) [reply test/SILGen/cf_curried_init.swift]: Worth noting that this failing on tvsimulator and `CG_AVAILABLE_STARTING(10.0, 2.0)` only defines availability for macOS and iOS. Not sure why this isn't failing on main, but I don't believe the check is important to this test. Allan is going to follow up.

**Comment by bnbarham** [comment]
@swift-ci please test

**Review by tshortli — APPROVED** [review]
Edit: accidentally left a stale comment here

**Comment by bnbarham** [comment]
Different test failures now 👍. Merging.

**Pull request merged**